src/Dapper.FSharp/MSSQL.fs

```diff
@@ -67,31 +62,24 @@ module private Evaluators =
         | { Take = None; Skip = x } when x <= 0 -> ""
         | { Take = None; Skip = o } -> sprintf "OFFSET %i ROWS" o
         | { Take = Some f; Skip = o } -> sprintf "OFFSET %i ROWS FETCH NEXT %i ROWS ONLY" o f
+    
+    let buildjoinType = function
+        | EqualsToColumn eqToCol -> (inBrackets eqToCol)
```

Comment:
The easiest thing to do would be to handle `| EqualsToConstant value -> (formatValue)` instead of parameterizing the join constant.
Where `formatValue` could be something like:

```F#
let formatValue (value: obj) = 
    match value with
    | :? string as str -> $"'{str}'"
    | :? DateTime as dt -> $"""'{dt.ToString("yyyy-MM-dd")}'"""
    | _ as o -> $"{o}"
```

That way you wouldn't have to parameterize it. 

In every case I can think of (off the top of my head at least), a join const would be something the developer would manually enter; whereas anything in the `where` clause would be like user input. 

Of course parameterizing would be the safest route, but more of a rabbit hole for sure.

Reply:
In such case I would leave it as it is now in this PR, because it works with parameters (as shown in the first example)